File: php-slim/lib/Model/RequestRoutesDepartureSearch.php
<?php
/**
 * RequestRoutesDepartureSearch
 */
namespace OpenAPIServer\Model;

/**
 * RequestRoutesDepartureSearch
 */
class RequestRoutesDepartureSearch
{
    
    /** @var string $id */
    private $id;
    
    /** @var string $departureLocationId */
    private $departureLocationId;
    
    /** @var string[] $arrivalLocationIds */
    private $arrivalLocationIds;
    
    /** @var \OpenAPIServer\Model\RequestTransportation $transportation */
    private $transportation;
    
    /** @var \DateTime $departureTime */
    private $departureTime;
    
    /** @var \OpenAPIServer\Model\RequestRoutesProperty[] $properties */
    private $properties;
    
    /** @var \OpenAPIServer\Model\RequestRangeFull $range */
    private $range;
}

File: fsharp-giraffe-server/OpenAPI/src/model/RequestRoutesDepartureSearch.fs
namespace OpenAPI.Model

open System
open System.Collections.Generic
open OpenAPI.Model.RequestRangeFull
open OpenAPI.Model.RequestRoutesProperty
open OpenAPI.Model.RequestTransportation

module RequestRoutesDepartureSearch = 

  //#region RequestRoutesDepartureSearch


  type RequestRoutesDepartureSearch = {
    Id : string;
    DepartureLocationId : string;
    ArrivalLocationIds : string[];
    Transportation : RequestTransportation;
    DepartureTime : DateTime;
    Properties : RequestRoutesProperty[];
    Range : RequestRangeFull;
  }
  //#endregion
  